### DISCUSSION TITLE
chiphell 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#9] [模板讨论]
chiphell 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by icaca** [comment]
Failed at 1/2 request,
Fail assert: {"re": "用户组", "from": "content"} from success_asserts,
Response Error : HTTP 567: Unknown,
Request URL: https://www.chiphell.com/forum.php
